.idea/libraries/org_eclipse_jetty_websocket_javax_websocket_server_impl_9_4_1_v20170120.xml

```diff
@@ -24,9 +24,9 @@
       <root url="jar://$MAVEN_REPOSITORY$/org/eclipse/jetty/jetty-client/9.4.1.v20170120/jetty-client-9.4.1.v20170120.jar!/" />
       <root url="jar://$MAVEN_REPOSITORY$/org/eclipse/jetty/websocket/websocket-common/9.4.1.v20170120/websocket-common-9.4.1.v20170120.jar!/" />
       <root url="jar://$MAVEN_REPOSITORY$/org/eclipse/jetty/websocket/websocket-api/9.4.1.v20170120/websocket-api-9.4.1.v20170120.jar!/" />
-      <root url="jar://$MAVEN_REPOSITORY$/javax/websocket/javax.websocket-api/1.0/javax.websocket-api-1.0.jar!/" />
```

Comment:
Replaced by version 1.1 -- which is already required by library `javax.websocket:javax.websocket-api:1.1`